app/components/chart-published.tsx

```diff
@@ -107,6 +108,11 @@ export const ChartPublished = (props: ChartPublishedProps) => {
   const { dataSource } = state;
   const locale = useLocale();
   const metadataPanelStore = useMemo(() => createMetadataPanelStore(), []);
+
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
+  const isDesktop = useMediaQuery(theme.breakpoints.up("lg"));
+  const isTablet = !isMobile && !isDesktop;
```

Comment:
Nit: We already use the `isMobile` logic in chart panel layout components; I think it would be cool to extract them to some utils file 🥹